kube/services/jupyterhub/jupyterhub-config.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jupyterhub-config
+data:
+  jupyterhub_config.py: |-
+    import os
+    c.JupyterHub.base_url = '/workspace'
+    c.JupyterHub.confirm_no_ssl = True
+    c.JupyterHub.db_url = 'sqlite:////etc/config/jupyterhub.sqlite'
+    c.JupyterHub.cookie_secret_file = '/etc/config/jupyterhub_cookie_secret'
+    c.JupyterHub.authenticator_class = 'jhub_remote_user_authenticator.remote_user_auth.RemoteUserAuthenticator'
+    #c.ConfigurableHTTPProxy.debug = True
+    c.JupyterHub.log_level = 'DEBUG'
+    c.JupyterHub.spawner_class = 'kubespawner.KubeSpawner'
+    c.KubeSpawner.namespace = 'jupyter-pods'
+    c.KubeSpawner.cpu_limit = 1.0
+    c.KubeSpawner.mem_limit = '1.5G'
+    #c.KubeSpawner.debug = False
+    c.KubeSpawner.notebook_dir = '/home/jovyan/pd'
+    c.KubeSpawner.singleuser_uid = 1000
+    c.KubeSpawner.singleuser_fs_gid = 1000
+    c.KubeSpawner.user_storage_pvc_ensure = True
+    c.KubeSpawner.user_storage_capacity = '10Gi'
+    c.KubeSpawner.pvc_name_template = 'claim-{username}{servername}'
+    c.KubeSpawner.user_storage_class = 'jupyter-storage'
+    c.KubeSpawner.volumes = [
+        {
+            'name': 'volume-{username}{servername}',
+            'persistentVolumeClaim': {
+                'claimName': 'claim-{username}{servername}'
+            }
+        }
+    ]
+    c.KubeSpawner.volume_mounts = [
+        {
+            'mountPath': '/home/jovyan/pd',
+            'name': 'volume-{username}{servername}'
+        }
+    ]
+    c.KubeSpawner.hub_connect_ip = 'jupyterhub-service.default'
```

Comment:
Do we need to hard code the '.default' namespace?
How is this going to work with namespaces for qa, dev, and red-black deployment?
This is actually a 3rd party package that we don't update as part of our core code - let's just move it out to it's own namespace rather than run in 'default' ...